core/.bazelrc

```diff
@@ -11,6 +11,7 @@ build --java_runtime_version=nixpkgs_java_11
 build --java_language_version=11
 build --tool_java_runtime_version=nixpkgs_java_11
 build --tool_java_language_version=11
+build --experimental_strict_java_deps=warn
```

Comment:
Could you add a short explanation here why this is necessary? And create a new issue which is linked in the comment please?

IIRC, this is only needed on macos, right? So, could we apply this only to this platform? (Maybe just leverage the [`--enable_platform_specific_config`](https://bazel.build/reference/command-line-reference#flag--enable_platform_specific_config) flag)

Reply:
Good call! I'll add this now.